bool red
bool yellow
bool green